keep/api/models/alert.py

```diff
@@ -393,12 +395,26 @@ def __str__(self) -> str:
     class Config:
         extra = Extra.allow
         schema_extra = IncidentDtoIn.Config.schema_extra
+        underscore_attrs_are_private = True
 
         json_encoders = {
             # Converts UUID to their values for JSON serialization
             UUID: lambda v: str(v),
         }
 
+    @property
+    def name(self):
+        return self.user_generated_name or self.ai_generated_name
+
+    @property
+    def alerts(self) -> List["AlertDto"]:
+        from keep.api.core.db import get_incident_alerts_by_incident_id
+        from keep.api.utils.enrichment_helpers import convert_db_alerts_to_dto_alerts
```

Comment:
this is here because it causes circular deps error?

Reply:
Yep, exactly

Reply:
would've been resolved if you'd just pass the alerts to the function :P